python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_functional_chaos.py

```diff
@@ -14,10 +14,10 @@
 # In this example we create a global approximation of a model response using
 # polynomial chaos expansion.
 #
-# Let :math:`h` be the function defined by:
+# Let :math:`\vect{g}` be the function defined by:
 #
 # .. math::
-#    g(\mathbf{x}) = \left[\cos(x_1 + x_2), (x_2 + 1) e^{x_1}\right]
+#    \vect{g}(\mathbf{x}) = \left[\cos(x_1 + x_2), (x_2 + 1) e^{x_1}\right]
```

Comment:
This is not exactly the same font:

![image](https://github.com/user-attachments/assets/f427b31d-ebfa-437a-a6cb-0b8a3d0be2cd)

```suggestion
#    \vect{g}(\vect{x}) = \Tr{\left(\cos(x_1 + x_2), (x_2 + 1) e^{x_1}\right)}
```